Src/VimCore/VimCore.fsproj

```diff
@@ -7,6 +7,7 @@
     <OtherFlags>--standalone</OtherFlags>
     <NoWarn>2011</NoWarn>
     <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+    <DebugType>portable</DebugType>
   </PropertyGroup>

```

Comment:
This ended up being required on newer versions of Visual Studio. It's unclear why but it doesn't affect our end to end developer experience, and portable PDBs are smaller, so just left it on.